certificate-policy.md

```diff
@@ -654,6 +660,10 @@ Certificate applications SHALL be processed and a certificate issued within 30 d
 ### 4.3.1 CA actions during certificate issuance
 Certificate issuance by the Root CA SHALL require an individual authorized by the CA (i.e. the CA system operator, system officer, or PKI administrator) to deliberately issue a direct command in order for the Root CA to perform a certificate signing operation.
 
+All end entity certificates asserting the id-kp-serverAuth EKU SHALL assert a Certificate Transparency (CT) Signed Certificate Timestamp (SCT) via a certificate extension. At a minimum, the CA SHALL request the following number of SCTs:
```

Comment:
is this wording accurate?  @weirdscience 

since it's based on validity period of the certificate, should this be aligned with the cert profiles and time period?
Also, did we agree on embedded SCT?  It was the conversation but there are three models.  

We had it in the cert profiles as embedded SCT - need to check if it was removed or still in there.